pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraCollectionPath.scala

```diff
@@ -122,14 +133,18 @@ private object HostEntryFieldCollectionPath
     val hostEntryField: Option[VarField] =
       bibData.varfieldsWithTag("773").find(_.subfieldsWithTag("w").nonEmpty)
 
-    if (hostEntryField.isDefined && bibId.nonEmpty)
-      Some(
-        CollectionPath(
-          path = collectionPathString(hostEntryField.get, bibId.trim),
-          label = None
-        )
-      )
-    else {
+    if (hostEntryField.isDefined && bibId.nonEmpty) {
+      collectionPathString(hostEntryField.get, bibId.trim) match {
+        case Some(path) =>
+          Some(
+            CollectionPath(
+              path = path,
+              label = None
+            )
+          )
+        case None => None
```

Comment:
Just my usual non-obligatory monadic nit-conjuring
```suggestion
      collectionPathString(hostEntryField.get, bibId.trim) map { path =>
        CollectionPath(
          path = path,
          label = None
        )
```

Reply:
Thanks, I always seem to miss this.  I start with a match because I want to do something different in the None case, then realise I don't need to, and never take that final step of changing it to map